Simplified/NYPLNetworkQueue.swift

```diff
@@ -8,7 +8,13 @@ import SQLite
  */
 final class NetworkQueue: NSObject {
 
-  static let shared = NetworkQueue()
+  static let sharedInstance = NetworkQueue()
+
+  // For Objective-C classes
+  @objc @discardableResult class func shared() -> NetworkQueue
+  {
+    return NetworkQueue.sharedInstance
```

Comment:
Same comment here as above. @winniequinn Your guidance on the proper usage in `didFinishLaunching..` would be appreciated!

Reply:
@gioneill + @vuinguyen: See above.

Reply:
@winniequinn my same question as above

Reply:
@gioneill How about I leave the discardableResult in this file, NYPLNetworkQueue.swift, and in Accounts.swift (above), and you can create a new issue / PR to remove them and take care of that?

Reply:
For now just delete `@discardableResult` in both places, and I'll address it in another PR. Thanks Vui!